CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ option(ENABLE_gdma "Enables Stone's GDMA multipole code (requires Fortran)" OFF)
 option(ENABLE_PCMSolver "Enables PCMSolver library (requires Fortran)" OFF)
 option(ENABLE_snsmp2 "Enables SNSMP2 plugin (can also be added at runtime)" OFF)
 option(ENABLE_v2rdm_casscf "Enables V2RDM_CASSCF plugin (can also be added at runtime)" OFF)
+option(ENABLE_gpu_dfcc "Enables gpu accelerated dfcc" OFF)
```

Comment:
`Enables gpu accelerated df-cc plugin (requires CUDA; can also be added at runtime)`

May as well add the `(requires Fortran;` to v2, too.